openapi-cli/src/test/java/io/ballerina/openapi/generators/schema/ConstraintTests.java

```diff
@@ -134,4 +134,46 @@ public void testAdditionalPropertiesWithConstraint() throws IOException, Balleri
     }
     //TODO current tool doesn't handle union type: therefore union type constraint will handle once union type
     // generation available in tool.
+
+    @Test(description = "Test for invalid constraint value")
+    public void invalidConstraintUses() throws IOException, BallerinaOpenApiException, FormatterException {
```

Comment:
Lets add the `test` prefix for all the new cases to be consistent with the old tests